pkgs/by-name/co/cosh/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, rustPlatform
+, rustc
+, fetchFromGitHub
+, openssl
+, pkg-config
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cosh";
+  version = "unstable-2024-01-27";
```

```suggestion
  version = "0.1-unstable-2024-01-27";
```
See https://github.com/NixOS/nixpkgs/blob/3c555dfb2b067689b4dccee0bcf046e76f4bb8ac/pkgs/README.md?plain=1#L353